README.md

```diff
@@ -90,18 +90,27 @@ test('Add new task', async t => {
 
 #### Obtaining component's props, computed and state
 
-In addition to [DOM Node State](http://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/dom-node-state.html), you can obtain `state`, `computed` or `props` of a Vue component. You can use them in an assertion directly thus simplifying assertion logic.
+In addition to [DOM Node State](http://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/dom-node-state.html), you can obtain `state`, `computed` or `props` of a Vue component.
+
 To get these data, use the Vue selector’s `.getVue()` method.
 
-If you call this method without parameters, it returns an object of the following structure.
+The `getVue()` method returns a [client function](https://devexpress.github.io/testcafe/documentation/test-api/obtaining-data-from-the-client.html). This function resolves to an object that contains component's properties, state and computed properties.
```

Comment:
> This function resolves to an object that contains component's properties, state and computed properties

This function resolves to an object that contains component's `props`, `state` and `computed` properties.

Reply:
Why property names here? I deliberately left the descriptions because we list the properties below

Reply:
Ok. Drop property names